### PR TITLE
Changes to the Medication Request Profile

### PR DESCRIPTION
**Pull request by SBajanki1**
NHSDigitalMedicationRequest profile changes

### CODE REVIEW

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml [hunk 1]
```diff
@@ -108,77 +124,68 @@
...
-      <definition value="Must only be populated if the  `requester` can not be responsible for the prescription, i.e. they would not be named as the prescriber on the FP10.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;extension&quot;: [&#xD;&#xA;        &#xD;&#xA;    {&#xD;&#xA;        &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner&quot;,&#xD;&#xA;        &quot;valueReference&quot;: {&#xD;&#xA;            &quot;reference&quot;: &quot;urn:uuid:a5acefc1-f8ca-4989-a5ac-34ae36741466&quot;,&#xD;&#xA;            &quot;display&quot;: &quot;DR SAZ RAZ&quot;&#xD;&#xA;        }&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;```" />
       <min value="0" />
       <max value="1" />
       <type>
         <code value="Extension" />
-        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ResponsiblePractitioner" />
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMResponsiblePractitioner" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
```
Comment (KazeemHamzat) [inline StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:133]: Any reason why the Must Support flag for Extensions was set to false?

File: MedicationRequest/Bundle-searchset-MedicationRequest.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
           },
           "extension": [
             {
-              "url": "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType",
+              "url": "https://fhir.nhs.uk/StructureDefinition/Extension-England-DMPrescriptionType",
               "valueCoding": {
                 "system": "https://fhir.nhs.uk/CodeSystem/prescription-type",
```
Comment (digitalhealthmartin) [inline MedicationRequest/Bundle-searchset-MedicationRequest.json:29]: This CodeSystem should be updated along with similar instances below.

File: StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml [hunk 3]
```diff
@@ -1,9 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="51c8f867-9fa0-4ca8-a82f-0f76d06d3bc2" />
-  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-ControlledDrug" />
-  <name value="ExtensionDMControlledDrug" />
+  <id value="Extension-England-DMControlledDrug" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMControlledDrug" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandDMControlledDrug" />
+  <title value="Extension England DM ControlledDru" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml:7]: *ControlledDrug

File: StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml [hunk 4]
```diff
@@ -1,9 +1,28 @@
...
   <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml:11]: Duplicate contact

File: StructureDefinition/Extension-DM-PrescriptionEndorsement.StructureDefinition.xml [hunk 5]
```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="a3d92f68-fc3d-4004-a5c9-b6eb6049d368"/>
-  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionEndorsement" />
-  <name value="ExtensionPrescriptionEndorsement" />
+  <id value="Extension-England-DMPrescriptionEndorsement" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMPrescriptionEndorsement" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandDMPrescriptionEndorsementNew" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-DM-PrescriptionEndorsement.StructureDefinition.xml:6]: Drop 'New' from name

File: StructureDefinition/Extension-DM-PrescriptionStatusHistory.StructureDefinition.xml [hunk 6]
```diff
@@ -29,10 +45,12 @@
         </discriminator>
         <rules value="open" />
       </slicing>
+      <min value="1" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-DM-PrescriptionStatusHistory.StructureDefinition.xml:48]: Should this cardinality be 1? Looks like this wasn't required previously

File: StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml [hunk 7]
```diff
@@ -1,9 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="8e2143ee-5d67-4731-9b23-6dc3aa72f9ae"/>
-  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType" />
-  <name value="ExtensionNHSBSAprescriptionType" />
+  <id value="Extension-England-DMPrescriptionType" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMPrescriptionType" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandNHSBSAPrescriptionType" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml:6]: Name and Title mention NHSBSA but not in ID. NHSBSA should be dropped from Name/Title to be consistent with URL/ID.

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml (hunk 1 above)
Comment (digitalhealthmartin) [reply StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:133]: Same point I have. Also why isModifier is set to false. Other instances throughout too.

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml [hunk 8]
```diff
@@ -108,77 +124,68 @@
...
       <min value="0" />
+      <max value="*" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:139]: Like other instances, * added as max cardinality but not sure if this is right or not.

File: StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml [hunk 9]
```diff
@@ -108,77 +124,68 @@
...
+        <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-EPSDispensingInformation" />
       </type>
-      <mustSupport value="true" />
+      <isModifier value="false" />
+    </element>
+    <element id="MedicationRequest.extension:NHSBSAPrescriptionType">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="NHSBSAPrescriptionType" />
+      <min value="1" />
+      <max value="1" />
```
Comment (digitalhealthmartin) [inline StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml:183]: Cardinality max to set to 1 but previously the max was not known so presumably would be * by default. Needs discussion.

File: fhirpkg.lock.json
Comment (digitalhealthmartin) [inline fhirpkg.lock.json:1]: This file should not have been included. This needs to be dropped.

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

File: StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml (hunk 3 above)
Comment (SheenaSharma) [reply StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml:7]: Replaced DM-ControlledDrug with ControlledDrug.

File: StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml (hunk 4 above)
Comment (SheenaSharma) [reply StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml:11]: Removed Duplicate Contact from this file.

File: StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml (hunk 3 above)
Comment (SheenaSharma) [reply StructureDefinition/Extension-DM-ControlledDrug.StructureDefinition.xml:7]: Did the changes as mentioned above.

**Comment by KazeemHamzat** [comment]
This PR is no longer required. Raise a new branch from DSTU2_develop to re-profile England MedicationRequest

File: StructureDefinition/Extension-DM-PrescriptionEndorsement.StructureDefinition.xml (hunk 5 above)
Comment (SheenaSharma) [reply StructureDefinition/Extension-DM-PrescriptionEndorsement.StructureDefinition.xml:6]: Removed 'New' from the name value.

File: StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml (hunk 7 above)
Comment (SheenaSharma) [reply StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml:6]: Did the required changes.

**Pull request closed without merging**